independent-publisher-connectors/Brønnøysundregistrene/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1608 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Br�nn�ysundregistrene",
+    "description": "",
+    "version": "1.0",
+    "contact": {
+      "name": "Ahmad Najjar",
+      "email": "ahmadn82@hotmail.com"
+    }
+  },
+  "host": "data.brreg.no",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/enhetsregisteret/api/enheter": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "_embedded": {
+                  "type": "object",
+                  "properties": {
+                    "enheter": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "organisasjonsnummer": {
+                            "type": "string",
+                            "description": "organisasjonsnummer"
```

Comment:
This change request needs discussion!

Well, as I said before this service is meant to be used in Norway only. These services (operations) and provided data are not relevant to anyone anywhere else.

Now here is catch no. 1, these services are used heavily all around Norway and it would be amazing to have them as an IP connector to facilitate the use of these services in automation (hence Power Automate) or showing the data in a unified and organized fashion (hence Power Apps).

Catch no. 2, I've worked with many that memorize these services, its operations, parameter summaries and descriptions like the back of their hands, translating them will definitely confuse most (if not all) consumers and won't provide any added value here. Additionally, as you can see from the swagger definition file, the services' schemas are defined mostly in Norwegian, though special characters are replaced as follows: 
- ø with o 
- å with a or aa
- æ with ae
That's why they look grammatically wrong.

Finally, as you may have noticed I have tried to follow the guidelines and write most stuff in English, so the the connector and its operations looks English from the outside, as for the returned dynamic content (data) I strongly recommend not to do that whereas this will decimate the whole idea of having this connector on the first place.

Let me know what you think @sriyen-msft

Reply:
@natalie-pienkowska Can you please help me how to proceed with this submission?

Reply:
@natalie-pienkowska and @mamurshe to provide approval for this PR

Reply:
@natalie-pienkowska and @mamurshe Ping

Reply:
@ahmadn82 We have discussed on your connector internally and given go ahead considering it as special case.